Katello/katello
# Fixes #27920 - refactor errata modeling to support pulp3

My steps to reproduce and test:
1. Download https://github.com/Katello/runcible/tree/master/test/fixtures/repositories/zoo5 and start HTTP server.
2. Sync this repo with katello running Pulp3
3. Sync repo https://fedorapeople.org/groups/katello/fakerepos/zoo2/
4. Notice that duplicate errata are created.

With this change, you should see no duplicate errata. Also if you update the local repo to add packages etc to the errata, you should see those getting added to the combined errata.
